### PR TITLE
Add Texture schema to Cesium 3D Metadata Spec

### PR DESCRIPTION
**Pull request by ptrgags**
This adds a couple missing schema files to the `schema` folder, and adds a few notes in the README to say that some examples are glTF specific (as `texture` is not actually part of this schema!)

@lilleyse could you review?

### CODE REVIEW

File: specification/Metadata/0.0.0/schema/textureAccessor.schema.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "A description of how to access data values from the color channels of a texture",
```
Comment (lilleyse) [inline specification/Metadata/0.0.0/schema/textureAccessor.schema.json:3]: `"title"` should just be "Texture Accessor" and this summary should go in a `"description"` property. The description should also include the note that the method of selecting a texture is implementation-defined.

File: specification/Metadata/0.0.0/schema/metadataTexture.schema.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Metadata Texture",
+    "type": "object",
+    "description": "Instances whose properties are stored directly in the texture designated by a class and property values stored in texture channels. This requires a class but not a feature table.",
```
Comment (lilleyse) [inline specification/Metadata/0.0.0/schema/metadataTexture.schema.json:5]: > This requires a class but not a feature table

I think this sentence can be removed

File: specification/Metadata/0.0.0/schema/metadataTexture.schema.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
```
Comment (lilleyse) [inline specification/Metadata/0.0.0/schema/metadataTexture.schema.json:6]: This line should be removed

File: specification/Metadata/0.0.0/README.md [hunk 4]
```diff
@@ -261,18 +261,21 @@ This is useful for continuous properties such as elevation, vegetation index, ve
...
+          }
         }
       }
     }
   }
 }
 ```
 
+**Note**: The method of selecting a texture is implementation-defined. The above example is based on a glTF implementation where `index` selects a texture and `texCoord` selects the texture coordinates. Other implementations may do this in
+alternate ways.
```
Comment (lilleyse) [inline specification/Metadata/0.0.0/README.md:277]:
```suggestion
**Note**: The method of selecting a texture is implementation-defined. The above example is based on a glTF implementation where `index` selects a texture and `texCoord` selects the texture coordinates. Other implementations may do this in alternate ways.
```

**Comment by ptrgags** [comment]
@lilleyse updated!

**Comment by lilleyse** [comment]
Thanks!

**Pull request merged**